pages/components/WidgetSize/ViewPort.js

```diff
@@ -1,20 +1,30 @@
 import styles from "./ViewPort.module.css"
 import { useState, useEffect } from "react"
 
-const ScreenWidth = () => {
+const ViewPort = () => {
   const [ width, setWidth ] = useState(0)
 
   const handleResize = () => {
     setWidth(window.innerWidth)
+    console.log("handleResize")
   }
 
-  useEffect(() => {
-    handleResize()
+  const onLoadHandler = () => {
     window.addEventListener("resize", handleResize)
```

Comment:
when you resize the window `handleResize` can be executed zillion times. Can you research how to avoid it?

Reply:
In the next version when it will be fixed. Changes are being prepared.